CRM/Upgrade/Incremental/php/FiveFortyFive.php

```diff
@@ -28,4 +28,24 @@ public function upgrade_5_45_alpha1($rev): void {
     $this->addTask('Update currency symbols for Belarus', 'updateCurrencyName', 'BYR', 'BYN');
   }
 
+  public function upgrade_5_45_2($rev): void {
+    $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+    $this->addTask('Install search kit if afform-admin is installed', 'installDependentExtension');
+  }
+
+  /**
+   * @param CRM_Queue_TaskContext $ctx
+   */
+  public function installDependentExtension(CRM_Queue_TaskContext $ctx) {
```

Comment:
```
---- CONSOLE OUTPUT: ----
Array
(
    [0] => CRM_Upgrade_Incremental_php_FiveFortyFive
    [1] => installDependentExtension
)
Exception: "Failed to call callback: 1"
#0 /home/jenkins/bknix-dfl/build/core-22595-5uivh/web/sites/all/modules/civicrm/CRM/Queue/Runner.php(201): CRM_Queue_Task->run(Object(CRM_Queue_TaskContext))
#1 /home/jenkins/bknix-dfl/build/core-22595-5uivh/web/sites/all/modules/civicrm/CRM/Queue/Runner.php(155): CRM_Queue_Runner->runNext()
```

The function should be static, shouldn't it?

```suggestion
  public static function installDependentExtension(CRM_Queue_TaskContext $ctx) {
```